### ISSUE TITLE
Add /data/pgsql/* to .gitignore

### ISSUE THREAD

**Issue by BramDriesen** [cytopia/devilbox#396]
The /data/pgsql/* directory is not in the .gitignore. which causes git to see it as a change and wants to push/commit the change.

**Comment by cytopia** [comment]
This is alredy there: https://github.com/cytopia/devilbox/blob/master/.gitignore#L21 or am I missing something?

**Comment by BramDriesen** [comment]
No, that's written with a Q /data/p**q**sql/* !== /data/p**g**sql/*

So either the Q is a typo and should be a G, or /data/pgsql/* needs to be added :)

**Comment by cytopia** [comment]
Nice catch. That went under the radar for a long time.

**Comment by cytopia** [comment]
Additionally I'd suggest to try out the upcoming branch, which will move all data (except www) to named volumes instead of directory mounts: https://github.com/cytopia/devilbox/pull/383
